R/multiEquatError.r

```diff
@@ -82,49 +93,56 @@ tripleEquatError <- function(e1, e2, e3, dependentDIF, testletStr, difBound, cal
   return(res)
 }
 
-checkInput <- function(inputlist) {

```

Comment:
I deleted this function, because the check in line 3 should be sufficient. When all arguments are a data frame, they are also of the same class, right? If not, I can change that back.

Reply:
That's right, good idea, thanks!